core/nginx-ingress/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
```

Comment:
Change this to kustomize.config.k8s.io/v1beta1

Reply:
We wanted to use flux one right?